secedgar/tests/utils/out.py

```diff
@@ -0,0 +1,1017 @@
+# Generated by gen_filing_type_enum.py as of 2/27/2021
```

Comment:
Why are you creating the filing types again here?

Reply:
see my comment